francisvicta-quezon.md

```diff
@@ -0,0 +1,3 @@
+## Quezon City, Philippines
```

Comment:
@caribmorpho It is. It is a little confusing, but there is also a city called Manila, which is part of Metro Manila.